scripts/bootstrap-master-4.pp

```diff
@@ -53,8 +53,8 @@
   manage_modulepath => false,
   sources           => {
     'gitlab' => {
-      'remote'  => 'https://github.com/genebean/control-repo.git',
-      'basedir' => "/etc/puppetlabs/code/environments",
+      'remote'  => 'https://github.com/thespain/control-repo.git',
```

Comment:
This will need to change back to my control repo. One way we _might_ be able to make this easier going forward would be to allow a custom fact to override the repo. If that fact exists then that path gets used, otherwise it stays with mine. Here's kinda the code idea to go with this:

```
mkdir -p /etc/puppetlabs/facter/facts.d
echo "control_repo_path: 'https://github.com/thespain/control-repo.git'" > /etc/puppetlabs/facter/facts.d/control_repo_path.yaml
```

With that idea, we could make the repo path block look like this (note I also made the source name make more since seeing as this isn't GitLab):

```puppet
if defined('$::control_repo_path') {
  $control_repo = $::control_repo_path
} else {
  $control_repo = 'https://github.com/genebean/control-repo.git'
}

class { 'r10k':
  provider          => 'puppet_gem',
  configfile        => '/etc/puppetlabs/r10k/r10k.yaml',
  manage_modulepath => false,
  sources           => {
    'vcs' => {
      'remote'  => $control_repo,
      'basedir' => '/etc/puppetlabs/code/environments',
      'prefix'  => false,
    }
  },
}
```